CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+
+Prerequisites:
+
+- Familiarity with [pull requests](https://help.github.com/articles/using-pull-requests) and [issues](https://guides.github.com/features/issues/).
+- Knowledge of [Markdown](https://help.github.com/articles/markdown-basics/) for editing `.md` documents.
+
+In particular, this community seeks the following types of contributions:
+
+- **Ideas**: participate in an issue thread or start your own to have your voice
+  heard.
+- **Resources**: submit a pull request to add to RESOURCES.md with links to related content.
+- **Outline sections**: help us ensure that this repository is comprehensive. if
+  there is a topic that is overlooked, please add it, even if it is just a stub
+  in the form of a header and single sentence. Initially, most things fall into
+  this category.
+- **Writing**: contribute your expertise in an area by helping us expand the included
+  content.
+- **Copy editing**: fix typos, clarify language, and generally improve the quality
+  of the content.
+- **Formatting**: help keep content easy to read with consistent formatting.
+
+# Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for
+all, regardless of gender, sexual orientation, disability, ethnicity, religion,
+or similar personal characteristic.
+
+On IRC, please avoid using overtly sexual nicknames or other nicknames that
+might detract from a friendly, safe and welcoming environment for all.
+
+Please be kind and courteous. There's no need to be mean or rude.
+Respect that people have differences of opinion and that every design or
+implementation choice carries a trade-off and numerous costs. There is seldom
+a right answer, merely an optimal answer given a set of values and
+circumstances.
+
+Please keep unstructured critique to a minimum. If you have solid ideas you
+want to experiment with, make a fork and see how it works.
+
+We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behaviour. We interpret the term "harassment" as
+including the definition in the
+[Citizen Code of Conduct](http://citizencodeofconduct.org/);
+if you have any lack of clarity about what might be included in that concept,
+please read their definition. In particular, we don't tolerate behavior that
+excludes people in socially marginalized groups.
+
+Private harassment is also unacceptable. No matter who you are, if you feel
+you have been or are being harassed or made uncomfortable by a community
+member, please contact one of the channel ops or any of the
+[CONTRIBUTING.md](https://github.com/jden/CONTRIBUTING.md) core team
+immediately. Whether you're a regular contributor or a newcomer, we care about
+making this community a safe place for you and we've got your back.
+
+Likewise any spamming, trolling, flaming, baiting or other attention-stealing
+behaviour is not welcome.
+
+# Communication
+
+There is an IRC channel on irc.freenode.net, channel `#CONTRIBUTING.md`. You're
```

Comment:
Please delete the IRC channel line, we don't have IRC, all the discussion takes place in form of issues. Thanks

Reply:
Sorry, I have remove the line